### PR TITLE
VICE Access Request admin processing

### PR DESCRIPTION
**Pull request by sri2k1us**
- Admin Processing Page

image

- Set Concurrent Job limit for approved request
image

- Reject access request
image

### CODE REVIEW

**Review by psarando — APPROVED** [review]
LGTM 👍 
I only had a couple of minor style suggestions 🎉 ✨

File: public/static/locales/en/vice-admin.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 {
+    "accessRequests": "Access Requests",
+    "showAllRequests": "Show All Requests",
+    "accessRequestApprovedMsg": "Your request to access VICE apps is approved.",
+    "accessRequestDeniedMsg": "We are unable approve your request to access VICE apps at this moment.",
```
Comment (ash3rz) [inline public/static/locales/en/vice-admin.json:5]: I was going to comment on how this message might be a little misleading, but I don't think it's actually used anywhere? 🤔

File: src/components/vice/admin/accessRequests/TableView.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                    const original = row.original;
+                    return (
+                        <IconButton
+                            color={
+                                original?.status?.toLowerCase() ===
+                                ACCESS_REQUEST_REJECTED
+                                    ? "primary"
+                                    : "default"
+                            }
+                            onClick={() =>
```
Comment (ash3rz) [inline src/components/vice/admin/accessRequests/TableView.js:108]: Having ids on the approve and deny buttons would be helpful for testing

**Review by ash3rz — APPROVED** [review]
LGTM :shipit:

File: public/static/locales/en/vice-admin.json (hunk 1 above)
Comment (sri2k1us) [reply public/static/locales/en/vice-admin.json:5]: Good catch. I added a feature of admins to add a custom instead of this one.

**Pull request merged**